.github/renovate.json

```diff
@@ -16,7 +16,7 @@
     "enabled": true
   },
    "ignorePaths": [
-    "/docs/**"
+    "**/docs/**"
```

Comment:
Weird, as docs is in the root folder, is n't it

Reply:
Yes indeed. 😅 Was also thinking about the heading `/` maybe that's the problem. Not sure.